ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs

```diff
@@ -0,0 +1,43 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using System.Collections.Generic;
+using Windows.UI.ViewManagement;
+
+namespace ReactNative.Modules.Accessibilityinfo
+{
+    class AccessibilityInfoModule : ReactContextNativeModuleBase
+    {
+        private readonly AccessibilitySettings _accessibility = new AccessibilitySettings();
+
+        public AccessibilityInfoModule(ReactContext reactContext)
+            : base(reactContext)
+        {
+            _accessibility.HighContrastChanged += (sender, args) =>
+            {
+                Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
+                    .emit("highContrastDidChange", sender.HighContrast);
+            };
```

Comment:
Can we set up and tear down this event in `Initialize` and `OnReactInstanceDispose`?

Reply:
Fixed